packages/composables/src/useUser/useUser.ts

```diff
@@ -264,6 +264,10 @@ export function useUser(): UseUserReturn {
   async function updatePersonalInfo(
     personals: operations["changeProfile post /account/change-profile"]["body"],
   ): Promise<void> {
+    if (personals.vatIds && !Array.isArray(personals.vatIds)) {
+      personals.vatIds = [personals.vatIds];
```

Comment:
We shouldn't correct the data here, component should send proper data and that's why it's typed

Reply:
But in that case, we should ask the question: Why do we need an API call in a composable? 
Removed